src/core/Sampler/Sampler.cpp

```diff
@@ -233,6 +236,229 @@ void Sampler::noteOff(Note* pNote )
 }
 
 
+// functions for pan parameters and laws-----------------
+
+float Sampler::getRatioPan( float fPan_L, float fPan_R ) {
+   /** returns the single pan parameter in [-1,1] from the L,R gains
```

Comment:
Could you move the doc string above the definition in order for Doxygen to find it? You can check using `./build.sh d`. And also in header file to not bloat the source code?